### PR TITLE
Fixed #26577 -- Disabled implicit wait of Selenium tests where appropriate.

### PR DESCRIPTION
**Pull request by charettes**

### CODE REVIEW

File: django/contrib/admin/tests.py [hunk 1]
```diff
@@ -147,28 +147,35 @@ def get_select_option(self, selector, value):
...
         Asserts that the <SELECT> widget identified by `selector` has the
         selected options with the given `values`.
         """
-        options = self.selenium.find_elements_by_css_selector('%s > option' % selector)
-        actual_values = []
-        for option in options:
-            if option.get_attribute('selected'):
-                actual_values.append(option.get_attribute('value'))
-        self.assertEqual(values, actual_values)
+        self._assertOptionsValues("%s > option:checked" % selector, values)
```
Comment (charettes) [inline django/contrib/admin/tests.py]: @dsanders11 I just realized that while there's no `:selected` selector `:checked` works just fine ([on IE 9+](http://caniuse.com/#search=%3Achecked))

**Comment by charettes** [comment]
Buildbot, test on selenium.

**Comment by timgraham** [comment]
I can't say for sure if this is the best solution but it seem okay for now at least.

**Comment by charettes** [comment]
I'll merge it as it seems to help Selenium tests to pass.

**Pull request merged**